Gui/typesystem_natronGui.xml

```diff
@@ -211,4 +211,8 @@
             </inject-code>
         </modify-function>
     </object-type>
+
+    <inject-code class="target" position="declaration">
+    NATRON_NAMESPACE_USING NATRON_PYTHON_NAMESPACE_USING
```

Comment:
Yes, and that's because during bindings generation the wrapped types are not under a namespace. It's much easier to do it here rather than in post-processing.